test/unit/spec/util/StringSpec.js

```diff
@@ -142,5 +142,35 @@ define(['util/string'], (StringUtil) => {
                 expect(StringUtil.isEmptyString(testCase)).toBeFalse();
             });
         });
+
+        [
+            ['abcd', 'abcd'],
+            ['ab_cd', 'ab_cd'],
+            ['ab cd', 'ab_cd'],
+            ['123', 'obj_123'],
+            ['ab&!*()', 'ab_____'],
+            ['a/b/c/abcd', 'a_b_c_abcd'],
```

Comment:
according to the replace expression at the top, the characters that are left as-is are:
`A-Z` `a-z` `0-9` `|` `.` `-` `_`

could you put in an example or two to cover all the characters that pass through the sanitising unscathed?

Reply:
Sure.

Reply:
Done!